napari/layers/_shapes_layer/shape_util.py

```diff
@@ -36,13 +66,13 @@ def inside_boxes(boxes):
 
     Parameters
     ----------
-    boxes : np.ndarray
-        Nx8x2 array of N boxes that should be checked
+    boxes : (N, 8, 2) array
+        Array of N boxes that should be checked
 
     Returns
     -------
-    inside : np.ndarray
-        Length N boolean array with `True` values for boxes containg the origin
+    inside : (N,) array of bool
+        Array with `True` values for boxes containg the origin
```

Comment:
I would take a page out of scikit-image's book:
```suggestion
        True if corresponding box contains the origin.
```

Reply:
fixed